addons/web/static/src/views/fields/properties/properties_field.js

```diff
@@ -672,9 +682,22 @@ export const propertiesField = {
         return {
             context: dynamicInfo.context,
             columns: parseInt(attrs.columns || "1"),
+            hideAddButton: archParseBoolean(attrs.hideAddButton),
             hideKanbanOption: archParseBoolean(attrs.hideKanbanOption),
         };
     },
 };
 
 registry.category("fields").add("properties", propertiesField);
+
+async function actionAddProperty(env) {
+    const addProperty = document.querySelector(".o_field_property_add button");
```

Comment:
that's the easiest way I found to do that

if we want to support many properties we can still pass the name of the field in the client action "params", but it add extra complexity (and because for now we have no model with 2 properties I did the simplest)